tests/conftest.py

```diff
@@ -5046,6 +5047,16 @@ def profiler_key(profiler_name: str) -> ConfigurationIdentifier:
     return ConfigurationIdentifier(configuration_key=profiler_name)
 
 
+@pytest.fixture(scope="function")
+def ge_cloud_profiler_id() -> str:
+    return "my_ge_cloud_profiler_id"
+
+
+@pytest.fixture(scope="function")
+def ge_cloud_profiler_key(ge_cloud_profiler_id: str) -> GeCloudIdentifier:
+    return GeCloudIdentifier(resource_type="contract", ge_cloud_id=ge_cloud_profiler_id)
+
+
```

Comment:
Necessary to test the GE Cloud specific code paths.

Reply:
@cdkini I affixed the following in #4111 for the RBP-relevant fixtures (right above the `@fixture`) decorator:
```
# TODO: <Alex>ALEX -- come up with a more elegant way of excluding all Rule-Based Profiler tests and fixtures, unless minimum Python version requirements are satisfied.</Alex>
@pytest.mark.skipif(sys.version_info < (3, 7), reason="requires Python3.7")
```

Thanks.

Reply:
Placed the same warnings - this will probably change since you merged first but I'll ensure parity for now